pkg/apis/stork/v1alpha1/backuplocation.go

```diff
@@ -176,6 +185,22 @@ func (bl *BackupLocation) UpdateFromClusterSecret(client kubernetes.Interface) e
 	return nil
 }
 
+func (bl *BackupLocation) getMergedNfsCred(client kubernetes.Interface) error {
```

Comment:
Can we rename this method to `getMergedNfsConfig`, just to align with other methods